demisto_sdk/commands/common/hook_validations/modeling_rule.py

```diff
@@ -1,28 +1,47 @@
 """
 This module is designed to validate the correctness of generic definition entities in content.
 """
-import logging
+import os
 
+from demisto_sdk.commands.common.errors import Errors
 from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
     ContentEntityValidator
+from demisto_sdk.commands.common.tools import get_files_in_dir
 
 
 class ModelingRuleValidator(ContentEntityValidator):
     """
     ModelingRuleValidator is designed to validate the correctness of the file structure we enter to content repo.
     """
 
+    def __init__(self, structure_validator, ignored_errors=None, print_as_warnings=False, json_file_path=None):
+        super().__init__(structure_validator, ignored_errors=ignored_errors, print_as_warnings=print_as_warnings,
+                         json_file_path=json_file_path)
+        self._is_valid = True
+
     def is_valid_file(self, validate_rn=True, is_new_file=False, use_git=False):
         """
         Check whether the modeling rule is valid or not
         Note: For now we return True regardless of the item content. More info:
         https://github.com/demisto/etc/issues/48151#issuecomment-1109660727
         """
-        logging.debug('Automatically considering XSIAM content item as valid, see issue #48151')
-        return True
+
+        self.is_schema_file_exists()
```

Comment:
That's good, let's also add a function that checks that the schema key in the yml itself is empty, so everyone will work with the new standard.
```suggestion
        self.is_schema_file_exists()
        self.is_schema_key_empty()
```

Reply:
In that effort, would you like me to check that the `rules` key in the yml also empty?